test/common_plat/validation/api/packet/packet.c

```diff
@@ -1405,7 +1405,7 @@ void packet_test_concat_small(void)
 
 	param.type    = ODP_POOL_PACKET;
 	param.pkt.len = len;
-	param.pkt.num = PACKET_POOL_NUM;
+	param.pkt.num = 65536;
```

Comment:
why?

Reply:
I agree with @muvarov. Magic numbers like this are a bad practice. Moreover, it's not clear why the validation test would need to change just because we're updating DPDK level.

Reply:
This PR is not ready yet for review. I by mistake put it on odp/cloud-dev instead of my private github. That's why I closed the PR. May be I should have written in the comment section.